python/tvm/relax/transform/legalize_ops/nn.py

```diff
@@ -177,6 +177,10 @@ def _nn_batch_norm(bb: BlockBuilder, call: Call) -> Expr:
         epsilon=call.attrs.epsilon,
         center=call.attrs.center,
         scale=call.attrs.scale,
+        # By default relax batch_norm is training mode.
+        # To transform it to eval mode, use Norm SimplifyInference pass.
```

Comment:
I suppose the name at this moment is SimplifyNormInference https://github.com/mlc-ai/relax/pull/120. Though it is not at the unity nor the mlc branch yet. Perhaps we can change the name now, and we'll have someone to commit that pass later.

Reply:
Yes I originally intend to change the formal name when upstreaming the pass (Since I'm not sure whether it will still be this name), but it's also ok I modify here